docs/user-tutorial/data-diagnosis.md

```diff
@@ -0,0 +1,142 @@
+---
+id: data-diagnosis
+---
+
+# Data Diagnosis
+
+## Introduction
+
+This tool is to filter the defective machines from thousands of benchmarking results automatically according to rules defined in **Rule file**.
+
+## Input
+
+The input mainly includes 2 files:
+
+ - **raw data**: jsonl file including multiple nodes' results automatically generated by SuperBench runner.
+
+    `Tips: the file can be found at ${output-dir}/results-summary.jsonl after each successful run.`
+
+ - **Rule file**: It uses YAML format and includes each metrics' rules to filter defective servers for diagnosis.
+
+- **baseline file**: json file including the baseline values for the metrics defined in the Rule file.
```

Comment:
````
json file including the baseline values for the metrics.
```
Would you please add your results repo here? It's as the tips.

Reply:
updated